openIt/jsonld-contexts/openIt.jsonld

```diff
@@ -3,6 +3,7 @@
     "Authorization": "https://vocab.egm.io/Authorization",
     "Gate": "https://vocab.egm.io/Gate",
     "Key": "https://vocab.egm.io/Key",
+    "category": "https://vocab.egm.io/category",
```

Comment:
use same URI as in https://github.com/easy-global-market/ngsild-api-data-models/blob/master/device/jsonld-contexts/device.jsonld#L9

Reply:
Done